uint32 leftSpeed
uint32 rightSpeed
uint32 leftDir
uint32 rightDir
